extension/src/popup/components/accountHistory/TransactionDetail/index.tsx

```diff
@@ -3,7 +3,6 @@ import { useSelector } from "react-redux";
 import StellarSdk from "stellar-sdk";
 
 import { Button } from "popup/basics/buttons/Button";
-import { PopupWrapper } from "popup/basics/PopupWrapper";
```

Comment:
`PopupWrapper` actually isn't super useful anymore. It was good when we had that bottom nav on a lot of the screens, but now it's just making the placement of the bottom button inconsistent :/